docs/integrations/sources/surveymonkey.md

```diff
@@ -1,72 +1,100 @@
+OAuth for Survey Monkey is supported only for the US. Airbyte is testing how to enable it in the EU. If you run into issues, [reach out to support](mailto:product@airbyte.io).
 
 :::
 
-<!-- env:oss -->
-
 ## Prerequisites
 
-**For Airbyte Open Source:**
+<Tabs groupId="platform">
+<TabItem value="cloud" label="Airbyte Cloud" default>
+
+<!-- env:cloud -->
+
+You have an [Airbyte Cloud](https://docs.airbyte.com/using-airbyte/getting-started/#sign-up-for-airbyte-clouds) account.
+
+
+<!-- /env:cloud -->
+
+</TabItem>
+
+<TabItem value="oss" label="Airbyte Open Source">
+
+<!-- env:oss -->
+
+You have installed [Airbyte](https://docs.airbyte.com/deploying-airbyte/quickstart).
 
-- Access Token
 <!-- /env:oss -->
 
+</TabItem>
+</Tabs>
+
 ## Setup guide
 
 ### Step 1: Set up SurveyMonkey
 
-Please read this [docs](https://developer.surveymonkey.com/api/v3/#getting-started). Register your application [here](https://developer.surveymonkey.com/apps/) Then go to Settings and copy your access token
+Read SurveyMonkey's [Getting started](https://developer.surveymonkey.com/api/v3/#getting-started). Register your app [at SurveyMonkey](https://developer.surveymonkey.com/apps/). Go to [My Apps](https://developer.surveymonkey.com/apps/) and click **Settings**. Copy your access token.
 
 ### Step 2: Set up the source connector in Airbyte
 
+<Tabs groupId="platform">
+<TabItem value="cloud" label="Airbyte Cloud" default>
+
 <!-- env:cloud -->
 
-**For Airbyte Cloud:**
+1. Log into your [Airbyte Cloud account](https://cloud.airbyte.com/workspaces).
+1. Click **+ New workspace** to create a new workspace. In the `Workspace name` field, enter a name, then click **Save changes**.
+1. In the left navigation bar, click **Sources**. Click **SurveyMonkey**, which takes you to the `Create a source` page.
+1. Click **Authenticate your account**.
+1. Log in to the SurveyMonkey account to authorize your app.
+1. In the `Start Date` field, set the required start date.
+1. Click **Set up source**.
+
+</TabItem>
 
-1. [Log into your Airbyte Cloud](https://cloud.airbyte.com/workspaces) account.
-2. In the left navigation bar, click **Sources**. In the top-right corner, click **+ new source**.
-3. On the source setup page, select **SurveyMonkey** from the Source type dropdown and enter a name for this connector.
-4. lick `Authenticate your account`.
-5. Log in and Authorize to the SurveyMonkey account
-6. Choose required Start date
-7. click `Set up source`.
 <!-- /env:cloud -->
 
+<TabItem value="oss" label="Airbyte Open Source">
+
 <!-- env:oss -->
 
-**For Airbyte Open Source:**
+1. Go to your [local Airbyte](http://localhost:8000/) page.
+1. In the left navigation bar, click **Sources**. Click **SurveyMonkey**, which takes you to the `Create a source` page.
+1. In the `Access Token` field, enter your SurveyMonkey access token from Step 1.
+1. In the `Start Date` field, set the required start date.
```

Comment:
Just a note that the UI is not consistent with the capitalization of the field headings. `Start Date` is title case while `Source name` is sentence case.